test/unit/lodash.spec.js

```diff
@@ -0,0 +1,709 @@
+'use strict';
+
+var chai = require('chai');
+var expect = chai.expect;
+var _ = require('lodash');
+var datasets = require('../../app/datasets');
+var Guy = require('./Guy');
+var chance = require('chance').Chance();
+
+
+
+describe('Lodash training Object', function ()
+{
+
+    describe('assign', function () {
+        var params;
+        beforeEach(function(){
+            params = datasets.assign();
+        });
+
+        it('should check types ', function(){
+            var elem1 = params[0];
+            var elem2 = params[1];
+
+            expect(typeof elem1).to.eql('object');
+            expect(elem2 instanceof Object).to.eql(true);
+        });
+
+        it('should check if first object has right properties', function(){
+            expect(_.assign(params[0])).to.have.ownProperty('number');
+            expect(params[0].number).to.equal(5);
+            expect(_.assign(params[0])).to.have.ownProperty('square', 25);
+            expect(_.assign(params[0])).to.have.ownProperty('ping');
+            expect(_.assign(params[0])).to.have.ownProperty('pong');
+        });
+        
+        it('should check if obj function (ping) return right value', function(){
+            expect(params[0].ping()).to.eql('Number: ' + params[0].number);
```

Comment:
You aren't checking if ping returns number that is inside it's context. I can as well hardcode answer right inside function.